extensions/amp-bind/0.1/test/validator-amp-bind.html

```diff
@@ -20,7 +20,7 @@
 <html ⚡>
 <head>
   <meta charset="utf-8">
-  <title>amp-bind: Basic</title>
+  <title>amp-bind</title>
```

Comment:
@Gregable looks like there was another PR in between the last revision bump and this rollup, #8457